test/elixir_analyzer_test.exs

```diff
@@ -150,6 +170,111 @@ defmodule ElixirAnalyzerTest do
     end
   end
 
+  describe "different failures" do
+    test "summary for a submission that did not run" do
```

Comment:
This is a bit artificial, because I needed to find a test that gives a report with an incomplete analysis, but could not really find a branching in the code that would lead to this. From the point of view of `Summary.summary/2` it makes sense to have that formatting option though...